roles/podman/tasks/main.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Variable configuration.
+- include_tasks: variables.yml
+
+# Setup/install tasks.
+- include_tasks: setup-RedHat.yml
+  when: ansible_os_family == 'RedHat'
+
+- name: Create directory for podman runtime config
+  ansible.builtin.file:
+    path: "~{{ podman_user }}/.config/containers"
```

Comment:
Is this really going to work if someone does the thing of making the user's home directory be elsewhere in the filesystem?  Do these Ansible built-in modules even do `~` expansion?

Reply:
tested by modifying instance install bundle playbook with 
```
    - name: Create the receptor user
      user:
        name: "{{ receptor_user }}"
        shell: /bin/bash
        home: "/var/lib/{{ receptor_user }}"
 ```
 
 ```
 TASK [Create the receptor user] ******************************************************************************************************************************************************************************************
changed: [remote-execution] => {"changed": true, "comment": "", "create_home": true, "group": 1001, "home": "/var/lib/awx", "name": "awx", "shell": "/bin/bash", "state": "present", "system": false, "uid": 1001}

...

TASK [ansible.receptor.podman : include_tasks] ***************************************************************************************************************************************************************************
included: /Users/liuhao/.ansible/collections/ansible_collections/ansible/receptor/roles/podman/tasks/variables.yml for remote-execution

...

TASK [ansible.receptor.podman : Create directory for podman runtime config] **********************************************************************************************************************************************
changed: [remote-execution] => {"changed": true, "gid": 1001, "group": "awx", "mode": "0700", "owner": "awx", "path": "/var/lib/awx/.config/containers", "secontext": "unconfined_u:object_r:container_var_lib_t:s0", "size": 6, "state": "directory", "uid": 1001}

TASK [ansible.receptor.podman : Configure podman default runtime] ********************************************************************************************************************************************************
changed: [remote-execution] => {"changed": true, "checksum": "2fee71581081ffcbcc68d2deb2309d5f6635d565", "dest": "/var/lib/awx/.config/containers/containers.conf", "gid": 1001, "group": "awx", "md5sum": "c35a9bef12a1c9557114140ff5739968", "mode": "0600", "owner": "awx", "secontext": "system_u:object_r:var_lib_t:s0", "size": 54, "src": "/home/ec2-user/.ansible/tmp/ansible-tmp-1665586239.6016672-50204-9527086907166/source", "state": "file", "uid": 1001}
```

Reply:
Huh, wow.